draft-ietf-webpush-protocol.xml

```diff
@@ -530,10 +558,11 @@ TTL = 1*DIGIT
       </t>
       <t>
         Each push message is pushed in response to a synthesized GET request.
-        The GET request is made to the push message resource that was created by
+        This GET request is made to the push message resource that was created by
         the push service when the application server requested message delivery.
-        The response body is the entity body from the most recent request sent
-        to the push resource.
+        The push service MUST include the link references to the push and receipt
+        subscribe resources in the request. The response body is the entity body

```

Comment:
Sorry for being slow this morning, but why does this need to include a receipts resource at all?